elastic/kibana
# [Infrastructure UI] Update host filter management to sync with AlertSummary

## 📓 Summary

Closes #153360 

To correctly sync the brush selection with the time service there was the need for a refactor on how we handle the host filters, simplifying a bit the code implementation and re-using existing abstractions.

## 🧪 Testing

- Navigate to Hosts View
- Create an Inventory Alert that can easily trigger
- Reload the search until the alerts are triggered
- Click on the alerts tab to preview them
- Select a time range on the chart and see the changes reflected on the chart and the time range globally applied.

https://user-images.githubusercontent.com/34506779/230015011-1b48deba-7c05-47df-80b1-37e8ee046b05.mov
